LICENSE.APL.txt

```diff
@@ -23,7 +23,7 @@ though please note that this license only applies to files which state in in the
 Apache License 2.0
 ++++++++++++++++++
 
-   Copyright 2020 PyInstaller Development Team
+   Copyright 2021 PyInstaller Development Team
```

Comment:
This one should be `2020-2021`, as it is actual copyright notice (contrary to the first one, which can indeed be seen as a template).